core/testMinimumPhpVersion.php

```diff
@@ -116,6 +116,7 @@ function Piwik_GetErrorMessagePage($message, $optionalTrace = false, $optionalLi
 
         if (!headers_sent()) {
             header('Content-Type: text/html; charset=utf-8');
+            header('Cache-Control: private, no-cache, no-store');
```

Comment:
Is there a reason for using multiple cacheability directives? Seems like we'd only need to use `no-store` (from reading https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#Cacheability).

Reply:
@diosmosis that's what we also use in production see eg https://stackoverflow.com/a/2068407/1447375 . In this case we don't set the Expires or Pragma no-cache but at least the Cache-Control.

Reply:
I see it's for different browsers. Fun.